bin/reth/src/main.rs

```diff
@@ -12,6 +11,9 @@ use reth_node_builder::{NodeBuilder, NodeConfig, NodeHandle};
 use reth_node_ethereum::EthereumNode;
 use reth_tasks::TaskManager;
 
+// Define a list of suffixes for chain IDs and RPC ports
+const SUFFIXES: [char; 2] = ['A', 'B'/* , 'C'*/]; // Add more suffixes as needed, like C
```

Comment:
Let's not have any arrays to define things, only generate stuff directly from the chain_id value, otherwise we can't easily support 100s of L2s.

Reply:
We do have to read some array of nodes the node runner wants to sync for from some config file